src/System.ServiceProcess.ServiceController/tests/ServiceProcessDescriptionAttributeTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.ServiceProcess.Tests
+{
+    public class ServiceProcessDescriptionAttributeTests
+    {
+        public static TheoryData<string> Ctor_Data => new TheoryData<string>
+        {
+            { string.Empty},
+            { null },
+            { "hello" }
+        };
+
+        [Theory,
+            MemberData(nameof(Ctor_Data))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Why skip on NETFX? It should pass there.

Reply:
@danmosemsft i think there is a bug in System.ComponentModel.Description attribute  in netfx because of which it is returning null in all three cases.